controllers/cloud.redhat.com/providers/utils/utils.go

```diff
@@ -33,6 +33,7 @@ var DefaultImageDatabasePG12 = "quay.io/cloudservices/postgresql-rds:12-2318dee"
 var DefaultImageDatabasePG13 = "quay.io/cloudservices/postgresql-rds:13-2318dee"
 var DefaultImageDatabasePG14 = "quay.io/cloudservices/postgresql-rds:14-2318dee"
 var DefaultImageDatabasePG15 = "quay.io/cloudservices/postgresql-rds:15-2318dee"
+var DefaultImageDatabasePG16 = "quay.io/cloudservices/postgresql-rds:16-f145533"
```

Comment:
Should the others be updated too??

Reply:
No need, main change was adding pgsql16 support. I am hoping to not trigger a DB update for all the others.